scraper/src/coordinates_scraper.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import os
+
+def start_processing():
+    data = fetch_data()
+    if (len(data) > 0):
+        transformed_data = transform_data(data)
+
+
+def fetch_data() -> list:
+    data = []
+    url = "https://life-api.coronasafe.network/data/oxygen_v2.json"
+    response = requests.get(url)
+    if ("data" in response.json() and len(response.json()["data"]) > 0):
+        data.extend(response.json()["data"])
+    return data
+
+def transform_data(data):
+    for record in data:
+        if (record):
+            address = ""
+            if "address" in record:
+                address += record["address"]
+            if "pincode" in record:
+                address += record["pincode"]
+            if len(address) > 0:
+                coordinates = fetch_coordinates(address)
+                if coordinates != None:
+                    record["lat"] = coordinates["lat"]
+                    record["lng"] = coordinates["lng"]
+    return data
+
+def fetch_coordinates(address):
+    url = generate_url(address)
+    response = requests.get(url)
```

Comment:
I don't believe we should be generating URL like this at all. I expect multiple issues with spaces, commas etc.

 params= {'address': address,
'key': os.environ["GEOCODE_KEY"],},

response = requests.get(
    url, params,
)

This is a much safer and recommended implementation.